### PR TITLE
doge house 🐕🏠...

### PR DESCRIPTION
**Pull request by Max-Hitchings**

### CODE REVIEW

File: 🐕.🐕 [hunk 1]
```diff
@@ -1 +1 @@
-🐕
+🐕🐕
```
Comment (goldyydev) [inline 🐕.🐕]: we're DogeHouse, not DogeDoge.
```suggestion
🐕🏠
```

**Review by goldyydev — CHANGES_REQUESTED** [review]

**Comment by mmattbtw** [comment]
You could also change the file name to `🐕.🏠`, if you wanted to.

**Review by vi241 — APPROVED** [review]

**Comment by vi241** [comment]
Merge? 👍 / 👎

**Review by vi241 — APPROVED** [review]

**Review by vi241 — APPROVED** [review]

**Comment by theCow61** [comment]
Originally the file was a gigabyte of doges but GitHub wouldn't let me push it because of a file limit. We either go big or small and because we can't go big with the gigabyte of 🐕 we should keep it small and simple with a singular 🐕 emoji in the file.

**Comment by vi241** [comment]
tg it didn't

**Comment by vi241** [comment]
that would be a big problem

**Pull request merged**